src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -849,13 +849,22 @@ export class BundleDataClient {
         // Keep track of fast fills that replaced slow fills, which we'll use to create "unexecutable" slow fills
         // if the slow fill request was sent in a prior bundle.
         const fastFillsReplacingSlowFills: string[] = [];
+
+        // Returns block number when we should start evaluating this fill for a refund. The lag builds in space for
+        // a filler to send a fill before a deposit in cases where the depositor is delegating transaction submission
+        // to the filler and giving them a signed transaction object.
+        const getFillBlockWithLag = (fill: V3FillWithBlock): number => {
+          const fillBuffer = /* FILL_BUFFER[fill.destinationChainId] ?? */ 0;
```

Comment:
I imagine to start we'd define a hardcoded value for this fill buffer. Technically it would need to be in the UMIP so it might even have to be a config store variable...